cmd/helm/installer/options.go

```diff
@@ -46,6 +46,9 @@ type Options struct {
 	// ServiceAccount is the Kubernetes service account to add to Tiller.
 	ServiceAccount string
 
+	// Force allows to force upgrading tiller if deployed version is greater than current version
+	ForceUpgrade bool
```

Comment:
Is there any value is making this more generic for re-use in other contexts with `helm init`?